Watching for file changes with StatReloader
Invalid HTTP_HOST header: '192.168.1.104'. You may need to add '192.168.1.104' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/x017/my-projects/nginx-service/twitter-clone/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/twitter-clone/venv/lib/python3.12/site-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/twitter-clone/venv/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/home/x017/my-projects/nginx-service/twitter-clone/venv/lib/python3.12/site-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '192.168.1.104'. You may need to add '192.168.1.104' to ALLOWED_HOSTS.
Bad Request: /
[29/Mar/2025 16:16:14] "GET / HTTP/1.0" 400 64207
/home/x017/my-projects/nginx-service/twitter-clone/core/settings.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 29, 2025 - 16:09:09
Django version 5.1.7, using settings 'core.settings'
Starting development server at http://0.0.0.0:8004/
Quit the server with CONTROL-C.

Watching for file changes with StatReloader
[29/Mar/2025 16:16:35] "GET / HTTP/1.0" 302 0
[29/Mar/2025 16:18:58] "GET / HTTP/1.0" 302 0
